css/main.css

```diff
@@ -44,6 +45,12 @@ h1 {
     font-size: 7em;
 }
 
+@media screen and (max-width: 490px) {
+    h1 {
+        width: 93%;
```

Comment:
Another options is we just reduce the font size? In fact we could just do that, below 490px reduce the font size to 2.5em or something? What you think?